### PR TITLE
Feature/fonenumber responses

### PR DESCRIPTION
**Pull request by calmdev**
Documentation updates for `/fonenumbers` routes.

https://dev-docs.fractel.net/

### CODE REVIEW

File: source/includes/_fonenumbers.md [hunk 1]
```diff
@@ -18,41 +18,29 @@ $ curl --request GET
...
-      "RewriteTo": [
-        ""
-      ]
+      "fonenumber": "8889807630",
+      "serviceType": "TimeOfDayRoute",
+      "receive": {
+        "url": "https://hookb.in/Ew7WzAb2",
+        "method": "JSON",
+        "username": "johndoe",
+        "password": "secret"
```
Comment (ubikorp) [inline source/includes/_fonenumbers.md]: Does the API actually return a password?

Comment (ubikorp) [reply source/includes/_fonenumbers.md]: This return data is completely different from the previous return data, I was wondering why I hadn't noticed that before.

File: source/includes/_fonenumbers.md [hunk 2]
```diff
@@ -18,41 +18,29 @@ $ curl --request GET
...
+      },
+      "sendNotify": {
+        "url": "https://hookb.in/Ew7WzAb2",
+        "method": "JSON",
+        "username": "johndoe",
+        "password": "secret"
+      },
+      "isActive": "yes"
     }
```
Comment (ubikorp) [inline source/includes/_fonenumbers.md]: Also noticing that incoming parameters are under_scored, while outgoing parameters are snakeCased.

File: source/includes/_fonenumbers.md (hunk 1 above)
Comment (calmdev) [reply source/includes/_fonenumbers.md]: Username and password properties should both be removed from the response.

Comment (calmdev) [reply source/includes/_fonenumbers.md]: I removed those, opened a PR and deployed an updated version of dev docs.

**Review by ubikorp — APPROVED** [review]

**Pull request merged**